hardware_accelerators/about-hardware-accelerators.adoc

```diff
@@ -0,0 +1,53 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="about-hardware-accelerators"]
+= About hardware accelerators
+include::_attributes/common-attributes.adoc[]
+:context: about-hardware-accelerators
+
+toc::[]
+
+Specialized hardware accelerators play a key role in the emerging generative artificial intelligence and machine learning (AI/ML) industry. Specifically, hardware accelerators are essential to the training and serving of large language and other foundational models that power this new technology. Data scientists, data engineers, ML engineers, and developers can take advantage of the specialized hardware acceleration for data-intensive transformations and model development and serving. Much of that ecosystem is open source, with a number of contributing partners and open source foundations.
+
+Red{nbsp}Hat OpenShift provides support for cards and peripheral hardware that add processing units that comprise hardware accelerators:
+
+* Graphical processing units (GPUs)
+* Neural processing units (NPUs)
+* Application-specific integrated circuits (ASICs)
+* Data processing units (DPUs)
+* Others
+
+image::OCP_HW_Accelerators_4.png[Supported hardware accelerators cards and peripherals]
+
+Specialized hardware accelerators provide a rich set of benefits for AI/ML development:
+
+One platform for all:: A collaborative environment for developers, data engineers, data scientists, and DevOps
+Extended capabilities with Operators:: Operators allow for bringing AI/ML capabilities to OpenShift
+Hybrid-cloud support:: On-premise support for model development, delivery, and deployment 
+Support for AI/ML workloads:: Model testing, iteration, integration, promotion, and serving into production as services
+
+Red{nbsp}Hat provides an optimized platform to enable these specialized hardware accelerators in {op-system-base-full} and OpenShift platforms at the Linux (kernel and userspace) and Kubernetes layers. To do this, Red{nbsp}Hat combines the proven capabilities of Red{nbsp}Hat OpenShift AI and Red{nbsp}Hat OpenShift in a single enterprise-ready AI application platform. 
+
+Hardware Operators use the operating framework of a Kubernetes cluster to enable the required accelerator resources. A device plugin is also provided that can be deployed manually or as a DaemonSet. This plugin enables the registration of the GPU in the cluster. 
```

Comment:
Suggesting for better readability / getting rid of passive voice. Also a suggestion in the second sentence, but not sure if that changes the meaning

```suggestion
Hardware Operators use the operating framework of a Kubernetes cluster to enable the required accelerator resources. You can also deploy the provided device plugin manually or as a daemon set. This plugin registers the GPU in the cluster. 
```